skatepark/block-templates/archive.html

```diff
@@ -1,35 +1,16 @@
 <!-- wp:template-part {"slug":"header-wide","tagName":"header"} /-->
 
-<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"top":"4em","bottom":"2.5em"}}},"layout":{"inherit":true}} -->
+<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"top":"4em","bottom":"2.5em"}}}} -->
 <main class="wp-block-group" style="padding-top:4em;padding-bottom:2.5em">
-	<!-- wp:query-title {"type":"archive","align":"wide"} /-->
-	<!-- wp:query {"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","categoryIds":[],"tagIds":[],"order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"exclude","inherit":false},"displayLayout":{"type":"flex","columns":3},"align":"wide"} -->
-	<div class="wp-block-query alignwide"><!-- wp:post-template -->
-	<!-- wp:group {"style":{"spacing":{"padding":{"top":"30px"}}},"layout":{"inherit":false}} -->
-	<div class="wp-block-group" style="padding-top:30px"><!-- wp:separator {"className":"is-style-wide"} -->
-	<hr class="wp-block-separator is-style-wide"/>
-	<!-- /wp:separator -->
-
-	<!-- wp:post-featured-image {"isLink":true} /-->
-
-	<!-- wp:post-title {"isLink":true,"fontSize":"normal"} /-->
-
-	<!-- wp:post-excerpt /-->
-
-	<!-- wp:post-date {"style":{"typography":{"fontSize":"var(--wp--custom--font-sizes--tiny)","fontWeight":"500"}},"isLink":true} /--></div>
+	<!-- wp:group {"layout":{"inherit":true}} -->
+	<div class="wp-block-group">
+		<!-- wp:query-title {"type":"archive","align":"wide"} /-->
+	</div>
 	<!-- /wp:group -->
-	<!-- /wp:post-template -->
-
-	<!-- wp:query-pagination -->
-	<!-- wp:query-pagination-previous /-->
-
-	<!-- wp:query-pagination-numbers /-->
-
-	<!-- wp:query-pagination-next /-->
-	<!-- /wp:query-pagination --></div>
-	<!-- /wp:query -->
+	
+	<!-- wp:template-part {"slug":"posts-grid"} /-->
```

Comment:
This is the same duplication that we have in most Blockbase themes since we removed the archive title tag from the index.html template. Is moving the duplicated code to a template part another example of creating additional complexity in the UI, or is it acceptable in this case?

cc @mtias

Reply:
I made it a pattern instead! We already had it in place, so we were duplicating code again. But after my conversation with @mtias over slack I think this is the way to go. He said we have to be careful mixing patterns and template parts, the difference being that template parts are assumed to be used somewhere. 

In this case, I think it makes more sense for it to be a pattern that we can just reuse over the theme and let the user reuse it too if they want as well. The pattern block sure is handy to have for this kind of stuff!